### ISSUE TITLE
Add `kmeans-colors` to Scoop Extras

### ISSUE THREAD

**Issue by oxjihun** [okaneco/kmeans-colors#61]
[Scoop](https://scoop.sh/) is a command-line installer for Windows. I wrote a following manifest that, once added to any of the Scoop's buckets such as [Extras](https://github.com/ScoopInstaller/Extras), would enable installing `kmeans-colors` via `scoop install kmeans-colors`.

``` json
{
    "version": "0.6.0",
    "description": "k-means clustering library and binary to find dominant colors in images",
    "homepage": "https://github.com/okaneco/kmeans-colors",
    "license": "MIT",
    "architecture": {
        "64bit": {
            "url": "https://github.com/okaneco/kmeans-colors/releases/download/0.6.0-binary/kmeans_colors-0.6.0-binary-windows-x86_64.tar.gz",
            "hash": "0d9060a8af1ebe19812618760b8e9f1f7907d925e523085b1a68d0ba9bcaa270"
        }
    },
    "bin": "kmeans_colors.exe",
    "checkver": "github",
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/okaneco/kmeans-colors/releases/download/$version-binary/kmeans_colors-$version-binary-windows-x86_64.tar.gz",
                "hash": {
                    "url": "https://github.com/okaneco/kmeans-colors/releases/download/$version-binary/kmeans_colors-$version-binary-windows-x86_64.sha256"
                }
            }
        }
    }
}
```

I am writing this issue to learn about how you might feel about this before actually making a PR there!

**Comment by okaneco** [comment]
Thanks, I just have one question about this.

When I update to a new major version like `0.7.0` or `1.0`, will the autoupdate tag pick up that as the new base version? Or will that require a manual update of the manifest to the next version? 

Otherwise, I'm generally in favor of letting more people use the tool if they find it helpful and the burden of maintenance seems minimal for me.

**Comment by oxjihun** [comment]
Yes, Scoop buckets have GitHub Actions that check for new versions (`0.7.0` or `1.0` would be correctly recognized as newer than `0.6.0`) and apply automatic updates to manifests according to the `autoupdate` fields. Manual updates would be needed only when new binaries are available or when there are alterations in naming schemes etc., but not for when new versions are released.

Thanks for positively responding, I will leave comments about how things go!

**Comment by oxjihun** [comment]
Ongoing PR: https://github.com/ScoopInstaller/Extras/pull/14624

**Comment by SorYoshino** [comment]
Hello, @okaneco .

I am a contributor to the [Extras bucket](https://github.com/ScoopInstaller/Extras).

When `kmeans-colors` was initially added to the Extras bucket, the download and autoupdate URLs in the manifest were based on tags whose `tag_name` contained `-binary`. This caused the auto-update mechanism to fail to update to releases whose `tag_name` does not include `-binary`, such as [0.7.1](https://github.com/okaneco/kmeans-colors/releases/tag/0.7.1) and [0.7.0](https://github.com/okaneco/kmeans-colors/releases/tag/0.7.0).

https://github.com/ScoopInstaller/Extras/blob/47037f140575000ebb83a099161037169853d18c/bucket/kmeans-colors.json#L14-L23

To allow the autoupdate mechanism to correctly update to the relevant versions, the associated URLs need to be updated. While I can modify the manifest to ensure that auto-updates are no longer affected by this issue, similar problems may still occur in the future in cases like having both [0.7.0](https://github.com/okaneco/kmeans-colors/releases/tag/0.7.0) and [0.7.0-binary](https://github.com/okaneco/kmeans-colors/releases/tag/0.7.0-binary).

I therefore suggest that future releases standardize the `tag_name`, for example by always using `$version` or always using `$version-binary`. This would reduce the likelihood of autoupdate failures and also make it easier for users to install a specific version using `scoop install kmeans-colors@<version>`.

Thank you very much for your contributions to this project!
